### PR TITLE
updates ember-cli 

### PR DESCRIPTION
**Pull request by jbailey4**
Updates ember-cli `2.13.2` -> `2.14.1`

Also, converts from `ember-computed-decorators` -> `ember-decorators`

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -25,15 +28,13 @@ matrix:
 before_install:
   - curl -o- -L https://yarnpkg.com/install.sh | bash
   - export PATH=$HOME/.yarn/bin:$PATH
-  - yarn global add bower phantomjs-prebuilt
-  - bower --version
```
Comment (alexdiliberto) [inline .travis.yml]: @jbailey4 Add back all the bower stuff

File: .travis.yml [hunk 2]
```diff
@@ -25,15 +28,13 @@ matrix:
...
+  - yarn global add phantomjs-prebuilt
   - phantomjs --version
 
 install:
-  - yarn install --no-lockfile
-  - bower install
```
Comment (alexdiliberto) [inline .travis.yml]: @jbailey4 
```js
  - bower install
```

Comment (alexdiliberto) [inline .travis.yml]: @jbailey4 Addressed in https://github.com/open-tux/ember-bulma/pull/70

File: .travis.yml (hunk 1 above)
Comment (alexdiliberto) [inline .travis.yml]: @jbailey4 Addressed in https://github.com/open-tux/ember-bulma/pull/70

File: .travis.yml (hunk 2 above)
Comment (jbailey4) [inline .travis.yml]: @alexdiliberto Lost track of this change during the upgrade. Good catch.

**Pull request merged**